docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,269 @@
+  large number of resources.  For example, the query ``netconn_count:[500 TO *]`` will match only processes that make
+  a large number of network connections.
+* When using ingress IOCs, be careful of errant characters in the ``values`` list, such as leading or trailing
+  whitespace or embedded newline characters.  These errant characters may cause the IOCs to fail to match, leading to
+  false negative results.
+* ``equality`` IOCs for IPv4 fields (e.g. ``netconn_remote_ipv4``) cannot support CIDR notation; full IP addresses
+  must be used.
+* ``equality`` IOCs for IPv6 fields (e.g. ``netconn_remote_ipv6``) do not support standard or CIDR notation at this
+  time. All IPv6 addresses must omit colon characters, spell out all zeroes in the address, and represent all
+  alphabetic characters in uppercase. For example, "ff02::fb" becomes "FF0200000000000000000000000000FB".
+
+Feeds
+-----
+Another way of managing reports is to attach them to a *feed.* Feeds can contain multiple reports, and a feed can be
+attached to a watchlist, effectively making the contents of the watchlist equivalent to the contents of the feed.
+
+Feeds are in effect “potentially-subscribable Watchlists”. A Feed has no effect on your organization until it is
+subscribed to, by creating a Watchlist containing that feed. Once subscribed (and until it’s disabled or unsubscribed),
+a watchlist will generate hits (and alerts if you have enabled them) for any matches against any of the IOCs in any of
+that feed’s enabled reports.
+
+**Note:** The feeds that are created by these examples are *private feeds,* meaning they are only visible within an
+organization and can be created by anyone with sufficient privileges in the organization.  There are additional types
+of feeds; *reserved feeds* can only be created by MSSPs, and *public feeds* can only be created or edited by
+VMware Carbon Black.
+
+A new feed may be created as follows (assuming the new report for that feed is stored in the ``report`` variable):
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Feed
+    >>> feed_data = {'feedinfo': {
+    ...                 'name': 'Suspicious Applications',
+    ...                 'provider_url': 'http://example.com/location',
+    ...                 'summary': 'Any signs of suspicious applications running on our endpoints',
+    ...                 'category': 'external_threat_intel',
+    ...                 'source_label': 'Where the info is coming from'},
+    ...              'reports': [report._info]}
+    >>> feed = api.create(Feed, feed_data)
+
+If you have an existing feed, a new report may be added to it as follows (assuming the new report is stored in the
+``report`` variable):
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Feed
+    >>> feed = cb.select(Feed, 'ABCDEFGHIJKLMNOPQRSTUVWX')
+    >>> feed.append_reports([report])
+
+To update or delete an existing report in a feed, look for it in the feed's ``reports`` collection, then call the
```

Comment:
This is good though we should also indicate `replace_reports` as an alternative for bulk operations because these will make a bunch of API calls if someone uses it in a for loop. The API is not very stable for this kind of operation

Reply:
I'm putting in a note about `replace_reports()`, which includes the need for caution as it replaces _all_ reports.